pallets/parachain-staking/src/lib.rs

```diff
@@ -595,7 +595,7 @@ pub mod pallet {
 			Self {
 				candidates: vec![],
 				delegations: vec![],
-				..Default::default()
+				inflation_config: Default::default(),
```

Comment:
@4meta5 this was producing a compile warning. Does this look good to you?

Reply:
yes nothing changes